src/query/service/src/test_utils/context.rs

```diff
@@ -23,16 +23,17 @@ use common_meta_app::principal::PasswordHashMethod;
 use common_meta_app::principal::UserInfo;
```

Comment:
`src/query/service/src/test_kits` is better than `src/query/service/src/test_utils`?